recipes/r-clang-activation/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "r-clang-activation" %}
+{% set version = "9.0.1" %}
+
+package:
+  name: "r_clang_{{ r_clang_target_platform }}"
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [not win]
+
+requirements:
+  run:
+    - "{{ compiler('clang') }}  {{ version }}"  # [win]
```

`compiler` macro has a version field, so using the suggestion below is future proof.
```suggestion
    - clang_{{ target_platform }}  {{ version }}
```